modules/virt-vmware-comparison.adoc

```diff
@@ -0,0 +1,52 @@
+
+:_mod-docs-content-type: CONCEPT
+
+[id="virt-vmware-comparison_{context}"]
+= Comparison of {VirtProductName} with VMware vSphere
+
+{VirtProductName} is conceptually different from VMware vSphere, much of its functionality comes from the underlying {product-title}, and it therefore does not offer direct alternatives to individual vSphere concepts and components. That said, if you are familiar with VMware vSphere, the following table lists the most common vSphere concepts and outlines what {VirtProductName} components you can use to accomplish a similar task.
+
+.Mapping of vSphere concepts to their closest {VirtProductName} counterparts
+[options="header"]
+[cols="2,2,3"]
+|===
+|vSphere concept |{VirtProductName} |Explanation
+
+|Datastore
+|Persistent volume (PV){nbsp}+ +
+Persistent volume claim (PVC)
+|Stores VM disks. A persistent volume represents existing storage and is attached to a VM through a persistent volume claim. When created with the `ReadWriteMany` (RWX) access mode, persistent volume claims can be mounted by multiple VMs simultaneously.
+
+|Dynamic Resource Scheduling (DRS)
+|Pod eviction policy{nbsp}+ +
+Descheduler
+|Provides active resource balancing. A combination of pod eviction policies and a descheduler allows VMs to be live migrated to more appropriate nodes to keep node resource utilization manageable.
+
+|NSX
+|Multus{nbsp}+ +
+OVN-Kubernates{nbsp}+ +
+Third-party container network interface (CNI) plug-ins
+|Provides overlay network configuration. There is no direct equivalent for NSX in {VirtProductName}, depending on the desired functionality, users can either rely on the OVN-Kubernates network provider, or install certified third-party CNI plug-ins.
+
+|Storage Policy Based Management (SPBM)
+|Storage class
+|Provides policy-based storage selection. Storage classes represent various storage types and describe storage capabilities such as quality of service, backup policy, as well as the reclaim policy or whether volume expansion is allowed. Persistent volume claims can request a specific storage class to satisfy the application requirements.
+
+|vCenter +
+vRealize Operations
+|OpenShift Metrics and Monitoring
+|Provides host and VM metrics. The {product-title} web console allows users to manage the VMs, view various metrics, and monitor the overall health of the cluster and VMs.
+
+|vMotion
+|Live migration
+|Allows the move of a running VM to another node without interruption. For live migration to be available, the persistent volume claim attached to the VM must have the `ReadWriteMany` (RWX) access mode.
+
+|vSwitch +
+DvSwitch
+|NMState Operator{nbsp}+ +
+Multus
+|Provides physical network configuration. The NMState Operator allows users to apply state-driven network configuration and manage various network interface types, including Linux bridges and network bonds. Multus allows users to attach multiple network interfaces and is often used to connect VMs to external networks.
```

Comment:
Should there be an article in the first sentence, like "a" or "the"?

This is good overall, but please consider my previous comment re: "allows," and the general guidance to use active voice instead of passive voice wherever possible.